build-system/tasks/prepend-global/index.js

```diff
@@ -176,8 +170,8 @@ async function applyConfig(
  * @param {!JSON} configJson The json object in which to enable local dev
  * @return {!JSON}
  */
-function enableLocalDev_(target, configJson) {
-  let LOCAL_DEV_AMP_CONFIG = {localDev: true};
+function enableTest_(target, configJson) {
+  let LOCAL_DEV_AMP_CONFIG = {test: true};
```

Comment:
nit: Rename this variable to `TEST_AMP_CONFIG` for better self-documentation.

Reply:
Done